docs/modules/operation/pages/deep-dive/admin/webui/grafana-dashboard-box.adoc

```diff
@@ -1,19 +1,21 @@
 
-
 [[webui-grafana-dashboard-box]]
 = Grafana Dashboard Box
 
-http://grafana.org/[Grafana] provides an API key which gives access for 3rd party application like {page-component-title}.
-The Grafana Dashboard Box on the start page shows dashboards related to {page-component-title}.
-To filter relevant dashboards, you can use a tag for dashboards and make them accessible.
-If no tag is provided all dashboards from Grafana will be shown.
-
-Configure this feature by creating or editing the `$\{OPENNMS_HOME}/etc/opennms.properties.d/grafana.properties` file.
+http://grafana.org/[Grafana] provides an API key that lets third-party applications access it.
+{page-component-title} uses Grafana to generate visualizations, including the Grafana Dashboard Box on the home page.
+By default, it shows a set of dashboards related to {page-component-title} and your monitored environment.
 
-.Quick access to Grafana dashboards from the {page-component-title} start page
+.Grafana dashboards on the {page-component-title} home page
 image::webui/startpage/01_grafana-box.png[]
 
-.Grafana Dashboard configuration properties
+== Filter dashboards
+
+To filter for relevant dashboards, you can use a dashboard's tag and make it accessible.
```

Comment:
Are there any examples of tags we can mention?

Reply:
I can look into this. :)